api/envoy/api/v2/core/grpc_method_list.proto

```diff
@@ -0,0 +1,24 @@
+syntax = "proto3";
+
+package envoy.api.v2.core;
+
+import "udpa/annotations/migrate.proto";
+import "validate/validate.proto";
+
+option java_package = "io.envoyproxy.envoy.api.v2.core";
+option java_outer_classname = "GrpcMethodListProto";
+option java_multiple_files = true;
+option (udpa.annotations.file_migrate).move_to_package = "envoy.config.core.v3";
+
+// A list of gRPC methods which can be used as a whitelist, for example.
```

Comment:
As long as we are doing new code/config can you s/whitelist/allowlist in this change to match current best practices in this area?